client/src/components/TinymceInlineToolbar/TinymceInlineToolbar.js

```diff
@@ -161,21 +161,41 @@ class TinymceInlineToolbar {
  * @return {Boolean}
  */
 function shouldShowToolbar(selection, isEditorFocused, tagTypes) {
-  const tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
-  return Boolean(
-    selection &&
-    tags
-      .map(tag => String(tag))
-      .map(tag => tag.toLowerCase())
-      .includes((selection.tagName || '').toLowerCase()) &&
-    isEditorFocused);
+  let tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
+
+  if (selection && isEditorFocused && Array.isArray(tagTypes) && tagTypes.length > 0) {
```

Comment:
we could return early with `if (!selection || !isEditorFocused) { return false; }`

I'm not sure about the `Array.isArray(tagTypes) && tagTypes.length > 0` part, this makes me think the `let tags` definition above redundant... Did you mean to make it `tags.length > 0` instead?